application/models/entities/Organisation_Model.php

```diff
@@ -0,0 +1,57 @@
+<?php
+defined('BASEPATH') or exit('No direct script access allowed');
+
+class Organisation_Model
+{
+
+    public String $name;
+    public String $bankAccountNumber;
+    public String $id;
+    public String $bankRefFormat;
+    public String $logoImg;
+    public String $tagline;
+    public String $supportEmail;
+
+    /**
+     * Constructor for organisation model
+     *
+     * @param String $name
+     * @param String $bankAccountNumber
+     * @param String $id
+     * @param String $bankRefFormat
+     * @param String $logoImg
+     * @param String $tagline
+     * @param String $supportEmail
+     */
+    function __construct($name, $bankAccountNumber, $id, $bankRefFormat, $logoImg, $tagline, $supportEmail)
+    {
+        $this->name = $name;
+        $this->bankAccountNumber = $bankAccountNumber;
+        $this->id = $id;
+        $this->bankRefFormat = $bankRefFormat;
+        $this->logoImg = $logoImg;
+        $this->tagline = $tagline;
+        $this->supportEmail = $supportEmail;
+    }
+
+    /**
+     *
+     * Setting organisation model variables to their specific element
+     *
+     * @param Array $elements
+     */
+    function array_Parameters($elements){
```

Comment:
Format this function code like the constructor.

The function could be better named to `toArray()` or something similar, since what we're essentially doing is converting the `Organisation_Model` class data into an array form.